tests/unit_tests/test_ladder_service.py

```diff
@@ -273,8 +274,9 @@ async def wait_forever(*args, **kwargs):
         mock.call({
             "command": "notice",
             "style": "info",
-            "text": "You have received 1 violations for failing to connect "
-            "to matchmaker games."
+            "text": "You have caused a matchmaking connection failure 1 time(s). "
+                    "Multiple failures result in temporary time-outs from matchmaker. "
+                    "Please seek support on the forums or discord for persistent issues."
```

Comment:
Here as well

Reply:
Added the brackets/format

Reply:
Aaah, sorry you didn't format it quite like I said. There should only be 1 additional level of indentation after the parens.

Reply:
Sorry hadn't realised the importance of indents in python (assumed it was just a code readability thing) - Is the updated commit what you had in mind?

Reply:
Yep that looks good! In this case it is just a code readability thing. We use one level of indentation everywhere in the code base so I'm just trying to keep it consistent.